knowagedao/src/main/java/it/eng/spagobi/utilities/JSError.java

```diff
@@ -79,7 +79,7 @@ public String toString() {
 	private static String getMessage(String key, String... args) {
 		String msg = MessageBuilderFactory.getMessageBuilder().getMessage(key);
 		if (args.length > 0) {
-			msg = MessageFormat.format(msg, args);
+			msg = String.format(Locale.getDefault(), msg, (Object[]) args);
```

Comment:
why using java.util.Locale.getDefault() ?
In my opinion this is a false positive, since msg is not really depending on user inputs